admin_guide/cluster_management.rst

```diff
@@ -145,3 +145,32 @@ The new shard(s) are created on the same node as the shard(s) from which the ten
     102240,
     'source_host', source_port,
     'dest_host', dest_port);
+
+Diagnostics
+###########
+
+.. _row_placements:
+
+Finding Row Placements
```

Comment:
How about something like "which shard contains data for a particular tenant?" That might be the FAQ question. The section here might be: "Finding which shard contains data for a specific tenant".

This is clearly more tenant specific, but that *might* be OK.